numpy/lib/mixins.py

```diff
@@ -5,53 +5,54 @@
 
 from numpy.core import umath as um
 
-# None of this module should be exposed in top-level NumPy module.
+# Nothing should be exposed in the top-level NumPy module.
 __all__ = []
 
 
+def _disables_array_ufunc(obj):
```

Comment:
Can you please explain the use-case you are envisioning in a little more detail? To be clear, you want to make it possible to subclass the mixin and override this check? For what end?

I am skeptical that making it easier for users to shoot themselves in the foot is a good thing :).